virtwho/main.py

```diff
@@ -157,6 +157,17 @@ def main():
             logger.error(err)
             exit(1, err)
 
+    if len(effective_config[VW_GLOBAL]['configs']) > 0:
+        cli_config_file_read = False
+        for file_name in effective_config[VW_GLOBAL]['configs']:
+            if os.path.isfile(file_name):
+                cli_config_file_read = True
+
+        if effective_config[VW_GLOBAL]['oneshot'] is True and cli_config_file_read is False:
+            err = 'No valid configuration file provided in one shot mode'
```

Comment:
I don't think that this should be limited to one shot. I know the bug says one shot, but it should apply regardless. If the user specifies a single config file and that file fails, then virt-who should stop.